index.html

```diff
@@ -1,12 +1,259 @@
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css"
+    />
+    <link rel="stylesheet" href="css/main.css" />
   </head>
-  <body></body>
+  <body class="body">
+    <header class="page_header">
+      <div>
+        <a href="/">
+          <img src="./images/Logo.svg" alt="CloudBudget" class="logo-img"
+        /></a>
+        <nav class="main-nav">
+          <ul class="ul-nav">
+            <li><a href="" class="li1">OVERVIEW</a></li>
+            <li><a href="" class="li1">FETURES</a></li>
+            <li><a href="" class="li1">TECHNOLOGY</a></li>
+            <li><a href="" class="li1">CONTACT</a></li>
+            <li><a href="" class="li1">SIGN UP</a></li>
+          </ul>
+          <button type="button" class="login_button">LOGIN</button>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="page_header">
+        <h1 class="main_head">CLOUDBUDGET</h1>
+        <p class="text_after_h1">
+          Cloud budget management for everyone. Only €69.95 a Month After a 7
+          Day Trial of Up to €4.99
+        </p>
+        <button class="sign_up_button">SIGN UP</button>
+      </section>
+      <section class="section_1">
+        <h2>Easy to Use Cloud Budget Management Software</h2>
+        <p class="text_section_1">
+          Our software is made so you can access and manage your budget and
+          expenses online at any time from any device. It provides detailed
+          income and expense reports with graphs so you can easilly see your
+          spending patterns and budget at a glance. Read below to find out more.
+        </p>
+        <div class="video-wrapper">
+          <video
+            src="./images/Eternal_Eclipse_Ithica.mp4"
+            class="video"
+            poster="./images/Eternal_Eclipse.jpg"
+            controls
+          ></video>
+          <button class="play-button">play</button>
+        </div>
+      </section>
+      <section class="section_2">
+        <h2>Features</h2>
+        <ul class="ul-nav">
+          <li>
+            <img
+              src="./images/picture_one.jpg"
+              alt="Зображення банкнот в руках людини"
+            />
+            <h3>Supports All Currencies and Cards</h3>
+            <p class="features_text">
+              We support all popular currencies and is fully customizable to add
+            </p>
+            <a href="" class="read_more">Read More</a>
+          </li>
+          <li>
+            <img
+              src="./images/picture_two.jpg"
+              alt="Мобільний телефон в руках дівчини"
+            />
+            <h3>Manage your expenses on the go</h3>
+            <p class="features_text">
+              You can access your account from anywhere in the world on any
+              device
+            </p>
+            <a href="" class="read_more">Read More</a>
+          </li>
+          <li>
+            <img src="./images/picture_three.jpg" alt="Склянний коридор" />
```

Comment:
I guess we should use only English for this project)

Reply:
Agree with you)